packages/razzle-plugin-typescript/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "env": {
+    "jest": true,
+    "node": true
+  }
+}
```

Comment:
I think we should actually move eslint out of here. I think we ought to move it of the core as well, making a razzle-plugin-eslint.

Reply:
Oh, this was just to make linting work on the code and test files. It won't be used by plugin users.